Bai-tap-Function-2/Bai4.py

```diff
@@ -0,0 +1,22 @@
+'''Bài 4: Viết hàm nhận vào n => In ra hình tam giác như Ví dụ.
+
+Input: func4(4)
+Output:
+****
+***
+**
+*
+'''
+
+def print_tri(m): 
+    x = m
+    arr = []
+    while x<=m and x>0: 
+        arr.append(x)
+        x -= 1
+    for i in arr:
+        print("*" * i,end ='')
+        print()
```

Comment:
@ngochan21098 
1. `x<=m` is unnessesary. Suggestion `while x<=m and x>0:` => `while x>0:`.

2. Another way is better:
```suggestion
    def print_tri(m): 
        for i in range(m):
            print("*" * (m - i), end ='')
            print()
```

Reply:
@ddhienpy I have understood your comment and update code here
https://github.com/ngochan21098/learning-python-basic/commit/258d14d70a1a8b45cb0ddeb15130b4df11cf3601